rocketpy/Environment.py

```diff
@@ -3221,6 +3221,117 @@ def allInfo(self):
 
         return None
 
+    def allPlotInfoReturned(self):
+        """Returns a dictionary with all plot information available about the Environment.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        ------
+        plotInfo : Dict
+            Dict of data relevant to plot externally
+        """
+        grid = np.linspace(self.elevation, self.maxExpectedHeight)
+        plotInfo = dict(
+            grid=[i for i in grid],
+            windSpeed=[self.windSpeed(i) for i in grid],
+            windDirection=[self.windDirection(i) for i in grid],
+            speedOfSound=[self.speedOfSound(i) for i in grid],
+            density=[self.density(i) for i in grid],
+            windVelX=[self.windVelocityX(i) for i in grid],
+            windVelY=[self.windVelocityY(i) for i in grid],
+            pressure=[self.pressure(i) / 100 for i in grid],
```

Comment:
This `/100` really bugs me.... Is there any reason for it to be here? Are the pressure values being returned in the dict in units different from Pa?

If so, I am not sure this is good practice, since RocketPy strives to use only Pa for consistency throughout the code.

Reply:
Since the ```allInfoReturned``` is based off the ```allInfo``` method, the pressure on the plots were given in hPa hence the ```/100``` but it can be implemented in hPa at the FE code and ```rocketpy``` stay in SI uniformly.

Reply:
You are right, we do plot in hPa to avoid scientific notation, which doesn't always look good on plots. Still, I am not sure storing information in hPa would be a good practice. 

I worry that there is no documentation behind it saying that it is in hPa, so this might cause some doubts for devs in the future.

Reply:
Totally comprehensible and I agree with you, I will make the changes necessary at FE and use only SI in my changes